### ISSUE TITLE
Only UI/UX?

### ISSUE THREAD

**Issue by wess** [app-developers/worldwide#2]
Is this top app developers for only UI/UX, not data or lower level frameworks?

**Comment by app-developers** [comment]
@wess Yes, now it is only list for UI/UX, we are glad that you pointed this out, as we plan to gather such selection in the future
